LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2016-2017 Alex Ellis
```

Comment:
This is because the code is imported. Let's add a second line `OpenFaaS Author(s)`